app/src/main/kotlin/io/imhungry/db/ImhungryDao.kt

```diff
@@ -1,6 +1,36 @@
 package io.imhungry.db
 
+import androidx.lifecycle.LiveData
 import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.Query
+import dagger.Component
+import io.imhungry.di.RoomModule
+import io.imhungry.model.RestaurantEntity
+import io.imhungry.model.RestaurantTagEntity
+import io.imhungry.model.TagEntity
 
 @Dao
-interface ImhungryDao
\ No newline at end of file
+interface ImhungryDao {
+
+    @Insert
+    fun insertTag(tag: TagEntity)
+
+    @Query("SELECT * FROM tag")
+    fun getAllTags(): LiveData<List<TagEntity>>
```

Comment:
to be more idiomatic to Kotlin, you should use something more like this:
```Kotlin
@get:Query("SELECT * FROM tag")
val allTags: LiveData<List<TagEntity>>
```
Let me know if you have problems implementing this.

This is just personal preference, feel free to argue against it.